src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  align-self: space-between;
+  width: 96px;
+  text-align: left;
+
+  &-star {
```

Comment:
`star` is an element of `stars` block. Please fix it so it's compliant to BEM.

Reply:
changes applied to the project.